packages/starpath/starpath.0.1.2/opam

```diff
@@ -0,0 +1,37 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A position-aware parser combinator library with readable errors"
+maintainer: ["Matthew Toohey <contact@mtoohey.com>"]
+authors: ["Matthew Toohey <contact@mtoohey.com>"]
+license: "BSD-3-Clause"
+tags: ["parser" "parser-combinators"]
+homepage: "https://github.com/mtoohey31/starpath"
+bug-reports: "https://github.com/mtoohey31/starpath/issues"
+depends: [
+  "ocaml" {>= "4.11"}
+  "dune" {>= "3.11"}
+  "ounit2"
```

Comment:
```suggestion
  "ounit2" {with-test}
```

Reply:
Thanks! I've updated things in my repo so the file will be generated this way in the future.